Microsoft.Toolkit.Uwp.UI/Extensions/Markup/OnDevice.cs

```diff
@@ -58,18 +62,7 @@ public class OnDevice : MarkupExtension
         /// <returns>The object value to set on the property where the extension is applied.</returns>
         protected override object ProvideValue()
         {
-            string deviceFamily = null;
-            if (DesignMode.DesignMode2Enabled)
```

Comment:
@michael-hawker , could we just ignore this?

Reply:
Yeah, we'd have to test this, I think the property isn't supported when queried in the designer, so not sure if we have a better solution here.

Reply:
My intent here was to remove an unused dependency (at least not yet used). We can still restore the design mode check once we know how to handle it ?